db/migrate/20210914084059_migrate_stripe_connect_enterprise_to_stripe_sca.rb

```diff
@@ -0,0 +1,6 @@
+class MigrateStripeConnectEnterpriseToStripeSca < ActiveRecord::Migration[6.1]
+  def up
+    execute "UPDATE spree_preferences SET key = replace( key, 'stripe_connect', 'stripe_sca') WHERE key LIKE '/spree/gateway/stripe_connect/'"
```

Comment:
This won't work without the `%` at the end of the pattern.

Reply:
Testing locally, I found a duplicate key violation because the stripe_sca key already existed. But that may just be my development data.